camel/agents/embodied_agent.py

```diff
@@ -120,7 +120,7 @@ def step(
 
         if codes is not None:
             content = "\n> Executed Results:"
-            action_space = {
+            action_space: Dict[str, Any] = {
```

Comment:
How does it work? Sometimes an action is a ToolAgent, sometimes a callable.

Reply:
If the input is a instance, all class member functions are available.